lib/internal/crypto/cipher.js

```diff
@@ -62,6 +62,13 @@ const { normalizeEncoding } = require('internal/util');
 
 const { StringDecoder } = require('string_decoder');
 
+const dc = require('diagnostics_channel');
+const onNewCipherChannel = dc.channel('crypto.cipher.new');
+const onNewDecipherChannel = dc.channel('crypto.decipher.new');
```

Comment:
`Cipher` and `Decipher` should not be used; I don't think it is beneficial to introduce any sort of monitoring for these APIs.

Reply:
I wasn't very sure about it. 
But as `cipher` and `decipher` are deprecated, `createCipher` and `createDecipher` methods are deprecated and defined like that, it is not easy to patch. 
```
ObjectDefineProperties(module.exports, {
  createCipher: {
    __proto__: null,
    enumerable: false,
    value: deprecate(createCipher,
                     'crypto.createCipher is deprecated.', 'DEP0106')
  }
```

But, if the rule is "not modify deprecated methods", I can delete this part of the code

Reply:
> But, if the rule is "not modify deprecated methods", I can delete this part of the code

There is no such rule; I just don't think adding complexity to a feature that nobody should be using might be the wrong approach.